src/types/mod.rs

```diff
@@ -150,11 +150,13 @@ macro_rules! pyobject_native_static_type_object(
 #[doc(hidden)]
 #[macro_export]
 macro_rules! pyobject_native_type_info(
-    ($name:ty, $typeobject:expr, $module:expr $(, #checkfunction=$checkfunction:path)? $(;$generics:ident)*) => {
+    ($name:ty, $typeobject:expr, $module:expr, $python_name:expr $(, #checkfunction=$checkfunction:path)? $(;$generics:ident)*) => {
         // SAFETY: macro caller has upheld the safety contracts
         unsafe impl<$($generics,)*> $crate::type_object::PyTypeInfo for $name {
             const NAME: &'static str = stringify!($name);
-            const MODULE: ::std::option::Option<&'static str> = $module;
+            const MODULE: ::std::option::Option<&'static str> = ::std::option::Option::Some($module);
+            #[cfg(feature = "experimental-inspect")]
```

Comment:
Need to be a bit careful here, the `cfg` does not exist for downstream crates. The right way to do this (painful) is to have separate definitions of the macro depending on whether the feature is enabled or not.